### ISSUE TITLE
Tor Browser vs. Tor Relay Node?

### ISSUE THREAD

**Issue by vitormilagres1992** [mynodebtc/mynode#263]
Hi everyone, this is not an issue, it's just a simple question, but I did not know where to ask it as I am new to Github.

When it says mynode has tor enabled by default, does that mean a tor relay node?
Or is it simply a tor browser?

I currently have a tor relay node running on my raspberry pi. I would be willing wipe it and fresh install a mynode instance on the same raspberry pi, as long as I am still able to keep my tor relay node running. 

In other words, when I install it on my raspberry pi, will the tor relay node be automatically setup up by default? If not, is it easy for me to enable a tor relay node instance while keeping all the benefits from mynode (bitcoin node + lightning + explorer...)?

**Comment by tehelsper** [comment]
On myNode Community, BTC and LND connect to their networks via Tor.

On Premium, Tor services are also launched to access Electrum, RTL, Web GUI, and many other services remotely via Tor.

**Comment by vitormilagres1992** [comment]
Oh great, so I guess all the connections will go through tor then.
Will I still be able to install a Tor Relay Node? I guess that would not be possible, right?

**Comment by tehelsper** [comment]
I'm not sure about the tor relay node. I'm sure you could install and run one, but it may conflict with the tor settings that could cause issues with upgrades, etc...
